packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -41,6 +41,13 @@
                         data-required
                     >
                         <oui-select-picker-section>
+                            <span
+                                class="oui-badge oui-badge_new new-flavor-badge"
```

Comment:
**nitpick:** Avoid custom css.
```suggestion
                                class="oui-badge oui-badge_new float-right position-relative"
```

Reply:
The bootstrap CSS class does not enable the definition of precise value for the `top` property. 
That why I had to use the custom CSS.